packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -683,7 +685,14 @@ const PostsPage = ({fullPost, postPreload, eagerPostComments, refetch, classes}:
             />
           </div>}
           <PostCoauthorRequest post={post} currentUser={currentUser} />
-          {!showSplashPageHeader && <PostsPagePostHeader
+          {!showSplashPageHeader && isBookUI && (showPostTitle !== "false") && <LWPostsPageHeader
```

Comment:
What's the `showPostTitle` logic?

Reply:
Oh, that was a temporary debugging thing. Removed.